src/main/java/com/jcabi/dynamo/mock/H2Data.java

```diff
@@ -170,8 +171,9 @@ public H2Data(final File file) {
     public Iterable<Attributes> iterate(final String table,
         final Conditions conds) throws IOException {
         try {
+            final String h2Table = this.encodeTableName(table);
```

Comment:
this is a one-time variable, please, avoid using it
see http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#one-time-variables for details